parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -639,6 +672,16 @@ public DataWriteBuilder withKeyMetadata(EncryptionKeyMetadata metadata) {
       return this;
     }
 
+    public DataWriteBuilder withFileEncryptionKey(ByteBuffer fileEncryptionKey) {
+      appenderBuilder.withFileEncryptionKey(fileEncryptionKey);
+      return this;
+    }
+
+    public DataWriteBuilder withAadPrefix(ByteBuffer aadPrefix) {
```

Comment:
This name was not updated to AAD. We need to have consistent method names throughout, so please double check.

Reply:
Sorry. Will run a case-sensitive search to update all methods.